integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseOpenUtils.java

```diff
@@ -112,13 +112,13 @@ public static OpenAction getOpenAction(int flag) {
   private static OpenAction getOpenActionFromExtraOpenFlags(int flag) {
     switch (flag) {
       case READ:
-      case READ_TWO:
       case READ_SYNC:
         return OpenAction.READ_ONLY;
       case WRITE:
         return OpenAction.WRITE_ONLY;
       case R_OR_W_PLUS:
       case RS_PLUS:
+      case READ_TWO:
```

Comment:
Can you rename this flag to READ_WRITE or R_OR_W?
and update the flag comments to say where you find it